.github/workflows/build.yml

```diff
@@ -0,0 +1,346 @@
+
+      - name: Setup | Publish Release
+        if: ${{ needs.release_please.outputs.release_created == 'true' }}
+        run: gh release edit ${{ needs.release_please.outputs.tag_name }} --draft=false --repo=lukso/lukso
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build | Add Artifacts to Release
+        uses: softprops/action-gh-release@v1
+        if: ${{ needs.release_please.outputs.release_created == 'true' }}
+        with:
+          files: lukso-*/lukso-*
+          tag_name: ${{ needs.release_please.outputs.tag_name }}
+
+  update_brew_formula:
+    name: Update Brew Formula
+    runs-on: ubuntu-latest
+    needs: [release_please, upload_artifacts]
+    # if: ${{ needs.release_please.outputs.release_created == 'true' }}
+    if: false
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v2.2
+        with:
+          formula-name: lukso
+          tag-name: ${{ needs.release_please.outputs.tag_name }}
+        env:
+          # Used for creating the formula update PR
+          COMMITTER_TOKEN: ${{ secrets.GH_PAT }}
+          # Used for verifying the SHA256 sum of the draft release
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+
+  publish_docs:
+    name: Publish script
+    needs: [notarize_and_pkgbuild, release_please]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup | Checkout
+        uses: actions/checkout@v3
+      - name: Copy script
+        if: ${{ needs.release_please.outputs.release_created == 'true' }}
+        run: |
+          mkdir -p dist
+          sed -e "s#__VERSION__#${{ needs.release_please.outputs.version_name }}" install/install.sh > dist/index.html
+      - name: Notarize | Download artifacts
+        uses: actions/download-artifact@v3
+        if: ${{ needs.release_please.outputs.release_created != 'true' }}
+        with:
+          path: artifacts
+      - name: Copy PR scripts
+        if: ${{ needs.release_please.outputs.release_created != 'true' }}
+        run: |
+          mkdir -p dist
+          rm -rf dist/artifacts
+          mkdir -p dist/artifacts
+          find artifacts -type f -exec cp {} dist/artifacts \;
+          sed \
+            -e 's#https://github.com/lukso-network/tools-lukso-cli/releases#https://lukso-network.github.io/tools-lukso-cli/pr-preview/pr-${{ github.event.number }}#g' \
+            -e 's#${BASE_URL}/latest/download#${BASE_URL}/artifacts#g' \
+            -e 's#__VERSION__#${{ needs.release_please.outputs.version_name }}#g' \
+            install/install.sh > dist/index.html
+      - name: Deploy
+        uses: JamesIves/github-pages-deploy-action@v4
+        if: ${{ needs.release_please.outputs.release_created == 'true' }}
+        with:
+          folder: ./dist
+          clean-exclude: pr-preview/
+      - name: Deploy PR
+        uses: rossjrw/pr-preview-action@v1
+        with:
+          source-dir: ./dist
```

Comment:
looks like copy/paste no? We don't need this here right?

Reply:
No this is correct the one is for pr previews and the other one for release